service/src/main/java/bio/terra/workspace/service/workspace/flight/GcpCloudSyncStep.java

```diff
@@ -54,11 +55,11 @@ public StepResult doStep(FlightContext flightContext)
       throws InterruptedException, RetryException {
     String gcpProjectId = flightContext.getWorkingMap().get(GCP_PROJECT_ID, String.class);
     FlightMap workingMap = flightContext.getWorkingMap();
-    // Read Sam groups for each workspace role. Stairway does not
-    // have a cleaner way of deserializing parameterized types, so we suppress warnings here.
+    // Read Sam groups for each workspace role.
     @SuppressWarnings("unchecked")
     Map<WsmIamRole, String> workspaceRoleGroupsMap =
-        workingMap.get(WorkspaceFlightMapKeys.IAM_GROUP_EMAIL_MAP, Map.class);
+        workingMap.get(WorkspaceFlightMapKeys.IAM_GROUP_EMAIL_MAP, new TypeReference<>() {});
```

Comment:
What happens if you remove the `@SuppressWarnings` here?

Reply:
They no longer appear... have removed them.